pennylane/registers.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the :func:`registers` function, which builds registers for a given dictionary
+of registers. Registers are a dictionary of :class:`~.Wires` objects, where the key is the name of
+the register and the value is the ``Wires`` object.
+"""
+
+from .wires import Wires
+
+
+def registers(register_dict):
+    """Returns a collection of wire registers when given a dictionary of register names and sizes
```

Comment:
How about this: """Returns registers of wires as a dictionary of register names and :class:`~pennylane.wires.Wires`.